========================================================================================
Report      : ERROR
Date        : Fri May 11 08:42:40 MST 2018
Description : dlopen of 'durability' failed: durability: cannot open shared object file: No such file or directory
Node        : m1m3testlap.lsst.org
Process     : python3.6 <17071>
Thread      : spliced 7f51ca738700
Internals   : 6.7.170523OSS///os_libraryOpen/os_library.c/70/0/1526053360.479157572/0
========================================================================================
Report      : ERROR
Date        : Fri May 11 08:42:40 MST 2018
Description : Problem opening 'durability'

Node        : m1m3testlap.lsst.org
Process     : python3.6 <17071>
Thread      : spliced 7f51ca738700
Internals   : 6.7.170523OSS///OpenSplice domain service/spliced.c/633/0/1526053360.479222760/0
========================================================================================
Context     : DDS::OpenSplice::FooDataWriter_impl::write
Date        : Fri May 11 12:28:38 MST 2018
Node        : m1m3testlap.lsst.org
Process     : python3.6 <17071>
Thread      : 7f512bfff700
Internals   : FooDataWriter_impl.cpp/232/6.7.170523OSS///0
----------------------------------------------------------------------------------------
Report      : Already deleted: Object is already deleted.
Internals   : DDS::OpenSplice::CppSuperClass::check/CppSuperClass.cpp/229/9/1526066918.682042161
========================================================================================
Report      : ERROR
Date        : Fri May 11 12:54:34 MST 2018
Description : dlopen of 'durability' failed: durability: cannot open shared object file: No such file or directory
Node        : m1m3testlap.lsst.org
Process     : python3.6 <20593>
Thread      : spliced 7ff6e12d5700
Internals   : 6.7.170523OSS///os_libraryOpen/os_library.c/70/0/1526068474.389860089/0
========================================================================================
Report      : ERROR
Date        : Fri May 11 12:54:34 MST 2018
Description : Problem opening 'durability'

Node        : m1m3testlap.lsst.org
Process     : python3.6 <20593>
Thread      : spliced 7ff6e12d5700
Internals   : 6.7.170523OSS///OpenSplice domain service/spliced.c/633/0/1526068474.413028000/0
========================================================================================
Context     : DDS::OpenSplice::FooDataWriter_impl::write
Date        : Fri May 11 12:55:21 MST 2018
Node        : m1m3testlap.lsst.org
Process     : python3.6 <20593>
Thread      : 7ff642ffd700
Internals   : FooDataWriter_impl.cpp/232/6.7.170523OSS///0
----------------------------------------------------------------------------------------
Report      : Already deleted: Object is already deleted.
Internals   : DDS::OpenSplice::CppSuperClass::check/CppSuperClass.cpp/229/9/1526068521.671408972
========================================================================================
Report      : ERROR
Date        : Fri May 11 12:55:51 MST 2018
Description : dlopen of 'durability' failed: durability: cannot open shared object file: No such file or directory
Node        : m1m3testlap.lsst.org
Process     : python3.6 <20715>
Thread      : spliced 7f5d4d481700
Internals   : 6.7.170523OSS///os_libraryOpen/os_library.c/70/0/1526068551.056436671/0
========================================================================================
Report      : ERROR
Date        : Fri May 11 12:55:51 MST 2018
Description : Problem opening 'durability'

Node        : m1m3testlap.lsst.org
Process     : python3.6 <20715>
Thread      : spliced 7f5d4d481700
Internals   : 6.7.170523OSS///OpenSplice domain service/spliced.c/633/0/1526068551.056493609/0
========================================================================================
Context     : DDS::OpenSplice::FooDataWriter_impl::write
Date        : Fri May 11 13:03:09 MST 2018
Node        : m1m3testlap.lsst.org
Process     : python3.6 <20715>
Thread      : 7f5caeffd700
Internals   : FooDataWriter_impl.cpp/232/6.7.170523OSS///0
----------------------------------------------------------------------------------------
Report      : Already deleted: Object is already deleted.
Internals   : DDS::OpenSplice::CppSuperClass::check/CppSuperClass.cpp/229/9/1526068989.320046507
========================================================================================
Report      : ERROR
Date        : Fri May 11 13:05:01 MST 2018
Description : dlopen of 'durability' failed: durability: cannot open shared object file: No such file or directory
Node        : m1m3testlap.lsst.org
Process     : python3.6 <20970>
Thread      : spliced 7f463bc42700
Internals   : 6.7.170523OSS///os_libraryOpen/os_library.c/70/0/1526069101.470358004/0
========================================================================================
Report      : ERROR
Date        : Fri May 11 13:05:01 MST 2018
Description : Problem opening 'durability'

Node        : m1m3testlap.lsst.org
Process     : python3.6 <20970>
Thread      : spliced 7f463bc42700
Internals   : 6.7.170523OSS///OpenSplice domain service/spliced.c/633/0/1526069101.470439272/0
========================================================================================
Report      : ERROR
Date        : Mon May 14 07:36:38 UTC 2018
Description : dlopen of 'durability' failed: durability: cannot open shared object file: No such file or directory
Node        : aclementsdell.lsst.org
Process     : python3.6 <17684>
Thread      : spliced 7f457cde6700
Internals   : 6.7.170523OSS///os_libraryOpen/os_library.c/70/0/1526283398.378239399/0
========================================================================================
Report      : ERROR
Date        : Mon May 14 07:36:38 UTC 2018
Description : Problem opening 'durability'

Node        : aclementsdell.lsst.org
Process     : python3.6 <17684>
Thread      : spliced 7f457cde6700
Internals   : 6.7.170523OSS///OpenSplice domain service/spliced.c/633/0/1526283398.378268987/0
========================================================================================
Context     : DDS::OpenSplice::FooDataWriter_impl::write
Date        : Mon May 14 07:37:56 UTC 2018
Node        : aclementsdell.lsst.org
Process     : python3.6 <17684>
Thread      : 7f4484ff9700
Internals   : FooDataWriter_impl.cpp/232/6.7.170523OSS///0
----------------------------------------------------------------------------------------
Report      : Already deleted: Object is already deleted.
Internals   : DDS::OpenSplice::CppSuperClass::check/CppSuperClass.cpp/229/9/1526283476.350999734
